src/Datadog.Trace.ClrProfiler.Managed/Integrations/StackExchange.Redis/ConnectionMultiplexer.cs

```diff
@@ -228,11 +229,29 @@ public static class ConnectionMultiplexer
 
         private static Scope CreateScope(object multiplexer, object message)
         {
-            var config = StackExchangeRedisHelper.GetConfiguration(multiplexer);
-            var hostAndPort = StackExchangeRedisHelper.GetHostAndPort(config);
-            var rawCommand = StackExchangeRedisHelper.GetRawCommand(multiplexer, message);
+            var multiplexerData = multiplexer.As<MultiplexerData>();
+            var hostAndPort = StackExchangeRedisHelper.GetHostAndPort(multiplexerData.Configuration);
+            var rawCommand = message.As<MessageData>().CommandAndKey ?? "COMMAND";
 
-            return RedisHelper.CreateScope(Tracer.Instance, IntegrationName, hostAndPort.Item1, hostAndPort.Item2, rawCommand);
+            return RedisHelper.CreateScope(Tracer.Instance, IntegrationName, hostAndPort.Host, hostAndPort.Port, rawCommand);
+        }
+
+        /*
+         * DuckTyping Types
+         */
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
```

Comment:
You could document these instead of disabling the warning 😉

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/985/commits/bf1cab2035da1c82eac4b7f68784feb8ad477565 :)